### PR TITLE
run shell commands one by one

### PR DESCRIPTION
**Pull request by ezkemboi**
- fixes issue #13

### CODE REVIEW

**Comment by ezkemboi** [comment]
@profnandaa as asked earlier, I have tried figuring out some of the possible ways. 
May be which modules should I make use of or any other approaches. 
Here is one of the **solutions (Not the best)**, but, it runs a loop for all the commands and returns messages each time to make the user sure that installation is on. 
If you have gotten any solution or library to use, do let me know, or we can see how this can be better. Thanks.

File: bin/installingdependancies.js [hunk 1]
```diff
@@ -27,42 +27,65 @@ const installingDependancies = (details) => {
...
+     * then, loop through each array 
+     * install each dependancy one by one
+     */
+    let cdBaseDirectory = `cd ${appBaseDirectory}`
+    const commands = [
+        `${cdBaseDirectory}`,
+        `set -e`,
+        `echo Will install your dependancies shortly....`,
+        `echo We are installing dependancies, please wait...`,
+        `${cdBaseDirectory} && npm install ${framework.toLowerCase()}`,
```
Comment (profnandaa) [inline bin/installingdependancies.js]: Tried having a temp `package.json` file with the deps defined there and just run npm install?

Comment (ezkemboi) [reply bin/installingdependancies.js]: What about how to manage versions of the same dependencies?

Comment (ezkemboi) [reply bin/installingdependancies.js]: Or any example file with the deps will be highly appreciated.

Comment (ezkemboi) [reply bin/installingdependancies.js]: > https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file 
I am checking on this line though still analyzing the best way

**Pull request closed without merging**